build.sbt

```diff
@@ -12,3 +12,15 @@ daffodilFlatLayout := true
 daffodilVersion := daffodilPackageBinVersions.value.head
 
 daffodilPackageBinVersions := Seq("3.5.0", "3.8.0", "3.7.0")
+
+daffodilPackageBinInfos := Seq(
+  DaffodilBinInfo("fakeTDL.dfdl.xsd", Some("fakeTDL")) // schema for a single message (in src)
+//  DaffodilBinInfo(
+//    "fakeTDLFile.dfdl.xsd",
+//    Some("fakeTDLFile")
```

Comment:
With this uncommented, you've provided two `DaffodilBinInfos`, with the same `name` parameter (i.e. the default of `None`). So the generated saved parsers would have the same name and so the artifacts would be the same. You need to provide a name for one of these so that the resulting saved parsers have different names, e.g.
```scala
daffodilPackageBinInfos := Seq(
  DaffodilBinInfo("/fakeTDL.dfdl.xsd", Some("fakeTDL")),
  DaffodilBinInfo("/fakeTDLFile.dfdl.xsd", Some("fakeTDLFile"), Some("file"))
)
```
Note that you could provide a `name` option for both if you want both saved parsers to have an additional name to make it clear which is which.